### DISCUSSION TITLE
How to improve the speed of paint image

### DISCUSSION THREAD

**Discussion by GISerliang** [emilk/egui#1227] [Q&A]
egui: 0.16.1
frame size: 1648x969
needed painted image size: 650x969

I used mesh like this:
```
// load data

// create mesh
let mut mesh = Mesh::default();
......
for row in 0..width {
  for col in 0..height {
    let rect = Rect::from_min_max(min_pos, min_pos.add(screen_size));
    let color = Color32::from_rgb(
                                        red_band as u8,
                                        green_band as u8,
                                        blue_band as u8,
                                    );
    mesh.add_colored_rect(rect, color);
  }
}
shapes.push(Shape::mesh(mesh));
```

load data duration time: 70ms
create mesh duration time: 400ms

One label will show mouse position when the mouse moved on frame, but the label text updated lag so much.
Could you help me?

**Comment by emilk** [comment]
Use `Context::load_texture` (on latest `trunk`) or `epi::Frame::alloc_texture` (0.16.0).
